# EuRoC V1_01_easy 双目相机 (monochrome) 参数配置

# 图像尺寸
image_width: 752
image_height: 480

# 左目相机内参与畸变
K_left: [458.654, 0.0, 367.215,
          0.0, 457.296, 248.375,
          0.0, 0.0, 1.0]
D_left: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05]

# 右目相机内参与畸变
K_right: [458.654, 0.0, 367.215,
           0.0, 457.296, 248.375,
           0.0, 0.0, 1.0]
D_right: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05]

# 旋转 R 从左相机到右相机
# R: [0.999992, 0.002673, -0.001653,
#     -0.002672, 0.999997, -0.000189,
#      0.001653, 0.000192, 0.999999]
R: [1.0, 0.0, 0.0,
    0.0, 1.0, 0.0,
    0.0, 0.0, 1.0]

# 平移 T 从左相机到右相机（单位：米）
T: [-0.110071, 0.000023, 0.000008]

